python-requre.spec

```diff
@@ -64,10 +65,10 @@ rm -rf html/.{doctrees,buildinfo}
 %{python3_sitelib}/%{pypi_name}
 %{python3_sitelib}/%{pypi_name}-%{version}-py%{python3_version}.egg-info
 
-%files -n python-%{pypi_name}-doc
```

Comment:
even more sneaky, no docs?

Reply:
It is regarding to https://bugzilla.redhat.com/show_bug.cgi?id=1796401 
reviewer said to adjust ownership in files, but I have no idea what that means, and after I've looked at maybe packit project there were no doc package, so I thought that this is what they think.